src/containers/Validators/validator.scss

```diff
@@ -6,20 +6,19 @@
 
   @include for-size(tablet-landscape-up) {
     width: 552px;
+    margin: auto;
   }
-
   @include for-size(desktop-up) {
     width: 772px;
   }
-
   @include for-size(big-desktop-up) {
     width: 1140px;
   }
 
   .loader {
     position: absolute;
     z-index: 1;
-    top: 0px;
+    top: 0;
```

Comment:
Why are you going from 0px to 0? (Is it better form to not include px for no padding?)

Reply:
better form to only use `0` instead of `0px`.